src/ModelIO/MDLNoiseTexture.cs

```diff
@@ -0,0 +1,35 @@
+#if XAMCORE_2_0 || !MONOMAC
+using System;
+using XamCore.ObjCRuntime;
+using Vector2i = global::OpenTK.Vector2i;
+
+namespace XamCore.ModelIO {
+
+	public enum  MDLNoiseTextureType {
+		Vector,
+		Cellular,
+	}
+
+	public partial class MDLNoiseTexture {
+		[iOS (9,0), Mac (10,11, onlyOn64 : true)]
+		public MDLNoiseTexture (float input, string name, Vector2i textureDimensions, MDLTextureChannelEncoding channelEncoding) : this (input, name, textureDimensions, channelEncoding, MDLNoiseTextureType.Vector)
+		{
+		}
+
+		public MDLNoiseTexture (float input, string name, Vector2i textureDimensions, MDLTextureChannelEncoding channelEncoding, MDLNoiseTextureType type)
```

Comment:
more like `[iOS (10,2), Mac (10,12, onlyOn64 : true)]` then, even if the API would partially work on earlier version